### PR TITLE
import test fix

### PR DESCRIPTION
**Pull request by alexcjohnson**
see https://github.com/plotly/dash/pull/1493

### CODE REVIEW

**Comment by alexcjohnson** [comment]
@xhlulu the tests show linter errors in code I didn't touch, care to take a look?

**Comment by xhlulu** [comment]
Sure gonna take a look!

**Comment by xhlulu** [comment]
For this PR 💃 

The problem will be fixed in this: https://github.com/plotly/dash-cytoscape/pull/116 I'll merge it after this one is merged.

**Review by xhlulu — APPROVED** [review]

**Pull request merged**
